lib/Conversion/TorchToLinalg/Utils.h

```diff
@@ -30,6 +30,12 @@ Value getPaddedTensor(Operation *op, OpBuilder &b, Value &input,
 Value getZeroPaddedTensor(Operation *op, OpBuilder &b, Value &input,
                           SmallVectorImpl<int64_t> &paddingInts);
 
+// Helper function that adds dynamic padding to a tensor, ignoring the first two
+// dimensions
```

Comment:
Is the `ignoring the first two dimensions` accurate? It seems that the ignored dimensions depends on `unpaddedDims`